tests.md

```diff
@@ -0,0 +1,228 @@
+
+# (Optionally) define the PHP executable to be called
+export WPT_PHP_EXECUTABLE=${WPT_PHP_EXECUTABLE-php}
+
+# (Optionally) define the PHPUnit command execution call. Use if `php phpunit.phar` can't be called directly for some reason.
+export WPT_PHPUNIT_CMD=
+
+# (Optionally) define the command execution to remove the test directory. Use if `rm -r` can't be called directly for some reason.
+export WPT_RM_TEST_DIR_CMD=
+
+# SSH connection string (can also be an alias). Leave empty if tests are meant to run in the same environment.
+export WPT_SSH_CONNECT=
+
+# Any options to be passed to the SSH connection. Defaults to '-o StrictHostKeyChecking=no'
+export WPT_SSH_OPTIONS=
+
+# SSH private key, base64 encoded.
+export WPT_SSH_PRIVATE_KEY_BASE64=
+
+# Output logging. Use 'verbose' to increase verbosity
+export WPT_DEBUG=
+```
+
+We will configure the folder where the WordPress software downloads and the database accesses will be made in order to prepare the tests.
+
+### Preparing the environment
+
+Before we do the first test, we'll catch up on everything. This process can be run before each test in this environment if we want to keep it up to date, although it will depend more if it is in a production environment.
+
+```bash
+cd /tmp/phpunit-test-runner/
+git pull
+npm update
+source .env
+```
+
+### Preparing the test
+
+Now that we have the environment ready, we can run the test preparation.
+
+```bash
+cd /tmp/phpunit-test-runner/
+php prepare.php
+```
+
+The system will run a long series of installations, configurations and compilations of different elements in order to prepare the test. If warnings and warnings come out you should not worry too much, as it is quite normal. At the end of the process it will warn you if it needs something it doesn't have. In principle, there should be something like this:
+
+```
+Done.
+Replacing variables in wp-tests-config.php
+Success: Prepared environment.
+```
+
+If the environment has been prepared, the next step is to make a first test.
+
+### Running the test
+
+Now that we have the environment ready, let's go get the test. To do this we will execute the file that will perform it.
+
+```bash
+cd /tmp/phpunit-test-runner/
+php test.php
+```
+
+What do the symbols mean?
+
+`.` → Each dot means that the test has been passed correctly.
+
+`S` → It means the test has been skipped. This is usually because these tests are only valid in certain configurations.
+
+`F` → Means that the test has failed. Information about why this happened is displayed at the end.
+
+`E` → It means that the test has failed due to a PHP error, which can be an error, warning or notice.
+
+`I` → Means that the test has been marked as incomplete.
+
+If you have followed all the steps up to this point it is very likely that the test you have run will fail quite a bit (for example, because by default it will not be able to do image processing). But that's OK, as I said, this is an example to check and verify the operation.
```

Comment:
I'd prefer not to suggest that tests will probably fail. Maybe explain that if they do, that helps find things that might help the environment be more compatible with WordPress?

Reply:
Changed to put it another way.